<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addUserDialog</class>
 <widget class="QDialog" name="addUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="usernameLabel">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="usernameLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="passwordLabel">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="passwordLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="fullNameLabel">
         <property name="text">
          <string>Full name</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="fullNameLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="sSNLabel">
         <property name="text">
          <string>SSN</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="sSNLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="administratorLabel">
         <property name="text">
          <string>Admin</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="administratorCheckBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Add user</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
